operations-stdlib/src/commonMain/kotlin/Length.kt

```diff
@@ -0,0 +1,12 @@
+import utils.asList
+import operation.StandardLogicOperation
+
+object Length : StandardLogicOperation {
+    override fun evaluateLogic(expression: Any?, data: Any?): Any? =
+        if (expression.asList.size > 1) null
```

Comment:
wdyt about reversed order:
`        if (expression.asList.size <= 1) {
            val element = expression.asList.firstOrNull()
            (element as? String)?.length
        } else null`
imo its easier to read what happens in the if statement